app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        elif visitor["wearing_a_mask"] is False:

```

Comment:
it's more pythonic way ;)
```suggestion
        elif not visitor["wearing_a_mask"]:

```